binder/plugins/views/csvexport.py

```diff
@@ -285,6 +285,11 @@ def get_datum(data, key, prefix=''):
 					else:
 						# Assume that we have a mapping now
 						fk_ids = data[head_key]
+
+						# If foreign key is null don't try to fetch the data and stop
```

Comment:
Can you also add this to the second bullet of the comment of the else statement?
https://github.com/CodeYellowBV/django-binder/pull/225/files#diff-4128d9ae74d08c29bcc4b2d5e871778087ec91853f76d1804283f78db3a9c97fR275-R278